src/m365/spo/commands/group/group-member-add.ts

```diff
@@ -73,6 +77,9 @@ class SpoGroupMemberAddCommand extends SpoCommand {
       },
       {
         option: '--email [email]'
+      },
+      {
+        option: '--userId [userId]'
```

Comment:
Could you validate if the userId option contains a numeric value or multiple comma separated values?

Also: I'm missing validations for the userName and email options. If you like, you can add those in this PR as well. Apparently someone forgot.